jslint4java-ant/src/test/java/com/googlecode/jslint4java/ant/PlainResultFormatterTest.java

```diff
@@ -1,7 +1,7 @@
 package com.googlecode.jslint4java.ant;
 
 import static org.hamcrest.CoreMatchers.*;
-import static org.junit.Assert.*;
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
What's the difference between the hamcrest assertThat and the JUnit assertThat?

(apologies I've not been in the Java ecosystem for some time)

Reply:
The old one is deprecated
https://junit.org/junit4/javadoc/4.13/org/junit/Assert.html#assertThat(T,%20org.hamcrest.Matcher)